en/docs/ai-gateway/ai-vendor-management/custom-ai-vendors/onboarding-anthropic-claude.md

```diff
@@ -1,16 +1,17 @@
-# Anthropic's Claude as a Custom AI Vendor
+# Onboarding Anthropic's Claude
 
 This guide explains how to integrate WSO2 API Manager with **Claude**, allowing you to access Claude's APIs via API Manager's AI Gateway. It provides step-by-step instructions for configuring Claude as a custom AI vendor to effectively manage and track AI API interactions.
 
-1. In the WSO2 API Manager admin portal sidebar, navigate to the **AI Vendors** section and select **Add AI Vendor**.
+1. Login to the Admin Portal
+2. Navigate to the **AI/LLM Vendors** section and click on **Add AI/LLM Vendor**.
 
     <div style="text-align: center;">
-        <a href="{{base_path}}/assets/img/administer/custom-ai-vendor/add-anthropic-claude/add-ai-vendor.png">
-            <img src="{{base_path}}/assets/img/administer/custom-ai-vendor/add-anthropic-claude/add-ai-vendor.png" width="600" />
+        <a href="{{base_path}}/assets/img/learn/ai-gateway/custom-ai-vendor/add-anthropic-claude/add-ai-vendor.png">
+            <img src="{{base_path}}/assets/img/learn/ai-gateway/custom-ai-vendor/add-anthropic-claude/add-ai-vendor.png" width="600" />
         </a>
     </div>
 
-2. Fill in the general details for the AI vendor as shown below.
+3. Fill in the general details for the AI vendor as shown below.
```

Comment:
Shall we fix the numbering after this? It is still 3,4,5 so 3 has been repeated twice.